app/configurator/components/add-dataset-dialog.tsx

```diff
@@ -138,6 +146,177 @@ const inferJoinBy = (
 
 export type JoinBy = ReturnType<typeof inferJoinBy>;
 
+export const PreviewDataTable: React.FC<{
```

Comment:
I think we shouldn't use `React.FC`, it's not recommended anymore even by React core team :) See e.g. https://github.com/facebook/create-react-app/pull/8177

Reply:
Ah it's done automatically by my refactoring extension, sorry 🙈 , thanks for the link !